homeassistant/components/tado/sensor.py

```diff
@@ -32,6 +32,7 @@
         "overlay",
         "early start",
         "open window",
+        "open window detected",
```

Comment:
```suggestion
```

Reply:
I am fine with this change, if you say that its OK when in one use-case the state-attrs are empty.